components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/dao/RoleDAOImpl.java

```diff
@@ -1190,11 +1190,12 @@ private void updateSharedSCIMRolesName(List<RoleDTO> sharedRoles, String newRole
             throws IdentityRoleManagementException {
 
         for (RoleDTO roleDTO : sharedRoles) {
+            String roleName = appendInternalDomain(roleDTO.getName());
```

Comment:
```suggestion
            String roleNameWithDomain = appendInternalDomain(roleDTO.getName());
```

Reply:
As all other places use this variable used the same.